remoting/src/test/java/org/apache/rocketmq/remoting/TlsTest.java

```diff
@@ -152,7 +153,9 @@ else if ("noClientAuthFailure".equals(name.getMethodName())) {
         remotingServer = RemotingServerTest.createRemotingServer();
         remotingClient = RemotingServerTest.createRemotingClient(clientConfig);
 
-        await().atMost(200, TimeUnit.MILLISECONDS).until(() -> isHostConnectable(getServerAddress()));
+        await()
+            .pollDelay(Duration.ZERO)
```

Comment:
Hi echooymxq, I do not see why specifying poll delay to zero fixes the issue, according to https://github.com/awaitility/awaitility/wiki/Usage#polling 

By default, the non-fixed poll interval is 0 at the beginning. 

Is there something I missed out?

Reply:
@lizhanhui Hi, zhanhui. Thank your review it, as your say, here is use defaultPollInterval and it  fixed poll interval, so it same poll delay as poll interval , it will sleep 100ms first and execute it. so the 200ms of tests offen cause timeout. The easiest way is to set it to 500ms, but i think there no need to wait it, because netty server channel bind is sync and tested work well.